### DISCUSSION TITLE
Preguntas sobre entrega MakeFile, tiempo

### DISCUSSION THREAD

**Discussion by Osiries1111** [IIC2133-PUC/2024-2#66] [Tarea 0] [answered]
Hola! tengo 2 preguntas.
- si cambio la optimización de MakeFile, tengo que subirlo al repositorio?
- Cuando hago time ./puffle input.txt output.txt, cual de los 3 tiempos se considerará para la evaluación? está el real, el sys y user.

**Comment by tomastrivino** [comment, marked as answer]
1.- si cambias la opción de optimización que sale en el Makefile, puedes subirlo a tu repositorio.
2.- real

**Comment by Osiries1111** [comment]
Respecto a 1. 
Qué optimización utilizarán a la hora de evaluar?

**Reply by sebacarrascop** [reply]
mientras solo cambies la optimización de tu makefile, se correrá con ese.
